lib/cli/index.js

```diff
@@ -0,0 +1,89 @@
+'use strict'
+const yargs = require('yargs/yargs')
+const path = require('path')
+const Loggerr = require('Loggerr')
```

Comment:
```suggestion
const Loggerr = require('loggerr')
```

Reply:
On linux the capital does not match the module name which has the lower case `l`

Reply:
Fixed!